public/components/trace_analytics/components/traces/span_detail_panel.tsx

```diff
@@ -48,16 +49,16 @@ export function SpanDetailPanel(props: {
   const [spanFilters, setSpanFilters] = useState<Array<{ field: string; value: any }>>(
     storedFilters ? JSON.parse(storedFilters) : []
   );
-  const [DSL, setDSL] = useState<any>({});
+  const [DSL, _setDSL] = useState<any>({});
```

Comment:
remove this? since it's no longer using DSL request

Reply:
Removed.